main/enums.py

```diff
@@ -16,7 +15,6 @@
     ('deployments', deployments_enums.enum_register),
     ('per', per_enums.enum_register),
     ('notifications', notifications_enums.enum_register),
-    ('databank', databank_enums.enum_register)
```

Comment:
We should keep this for now.

Basically keep the link between each app and global enums.